.github/workflows/quickstart_http-custom-mechanism_ci.yml

```diff
@@ -0,0 +1,17 @@
+name: WildFly http-custom-mechanism Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - http-custom-mechanism/**'
```

Comment:
- 'http-custom-mechanism/**'

Reply:
Sorry about that. It is fixed now. 
Thanks for pointing it out!